### PR TITLE
longname for update_salt_bootstrap function changed to update_bootstrap

### PR DESCRIPTION
**Pull request by inthecloud247**

### CODE REVIEW

File: saltcloud/utils/parsers.py [hunk 1]
```diff
@@ -199,7 +199,7 @@ def _mixin_setup(self):
             help='Build all of the specified virtual machines in parallel'
         )
         group.add_option(
-            '-u', '--usb',
+            '-u', '--update_bootstrap',
```
Comment (techhat) [inline saltcloud/utils/parsers.py]: This would need to be --update-bootstrap, not --update_bootstrap.

**Comment by inthecloud247** [comment]
k this should be good now.

**Comment by techhat** [comment]
You're going to want to start using feature branches in git, to keep those merge commits out.

**Comment by inthecloud247** [comment]
yeah just noticed that... :-) thanks!

**Pull request merged**